server/src/service.rs

```diff
@@ -114,16 +114,61 @@ impl Service {
     }
 
     #[zbus(out_args("collection"))]
-    pub async fn read_alias(&self, _name: &str) -> Result<ObjectPath, ServiceError> {
-        todo!()
+    pub async fn read_alias(
+        &self,
+        name: &str,
+        #[zbus(object_server)] object_server: &zbus::ObjectServer,
+    ) -> Result<ObjectPath, ServiceError> {
+        let mut objectpath = ObjectPath::default();
+        for collection in self.collections.lock().await.iter() {
+            let collection_ifce_ref = object_server.interface::<_, Collection>(collection).await?;
+            let collection = collection_ifce_ref.get_mut().await;
+
+            if collection.alias().await == name || collection.label().await == name {
+                objectpath = ObjectPath::from(collection.path().clone());
+            }
+        }
+
+        Ok(objectpath)
     }
 
     pub async fn set_alias(
         &self,
-        _name: &str,
-        _collection: ObjectPath<'_>,
+        name: &str,
+        collection: ObjectPath<'_>,
+        #[zbus(object_server)] object_server: &zbus::ObjectServer,
     ) -> Result<(), ServiceError> {
-        todo!()
+        if name != "default" {
+            return Err(ServiceError::ZBus(zbus::Error::Failure(String::from(
+                "Only the 'default' alias is supported",
+            ))));
+        }
```

Comment:
I don't think it makes sense to re-add the same restrictions as gnome-keyring, the goal is to be better than gnome-keyring.

Reply:
Yeah I agree. I think we should get to gnome-keyring level first, then we can add more better/cool stuff later :)

Reply:
This condition doesn't affect anything anyways, so please drop it. As my goal is not to just be compatible with gnome-keyring but with spec and the spec says nothing about this restriction.

Reply:
Removed.

Note: I think KWallet does the same thing: https://invent.kde.org/frameworks/kwallet/-/blob/master/src/runtime/kwalletd/kwalletfreedesktopservice.cpp#L776

Reply:
No it doesn't error out, it just sets the information for the default collection, nothing more.

Reply:
Ah ok, thanks!